scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -36,7 +36,6 @@ import io.mockk.mockk
 import io.mockk.mockkStatic
 import io.mockk.spyk
 
-import java.io.File
```

Comment:
Odd. Any idea why `detekt` didn't complain in the first place?

Reply:
Ah, it probably got confused with `org.ossreviewtoolkit.clients.fossid.model.identification.identifiedFiles.File` / `org.ossreviewtoolkit.clients.fossid.model.identification.markedAsIdentified.File`.